runtime/ts/storage/firebase-storage.ts

```diff
@@ -117,6 +117,12 @@ export class FirebaseStorage extends StorageBase {
   }
 
   async connect(id: string, type: Type, key: string) : Promise<FirebaseStorageProvider> {
+    let referenceMode = !(type instanceof ReferenceType);
```

Comment:
Why do we have to do this FB-specific logic here for a Pouch PR?

Reply:
I removed this and switched back to an explicit _join for now.  At some point we should be able to use the public APIs for firebase.